org.uqbar.project.wollok/src/org/uqbar/project/wollok/validation/WollokDslValidator.xtend

```diff
@@ -1135,6 +1135,14 @@ class WollokDslValidator extends AbstractConfigurableDslValidator {
 			report(WollokDslValidator_DONT_DUPLICATE_TEST_DESCRIPTION, wtest, WTEST__NAME)
 	}
 
+	@Check
+	@DefaultSeverity(WARN)
+	@CheckGroup(WollokCheckGroup.POTENTIAL_DESIGN_PROBLEM)
+	def testWithEmptyDescription(WTest t) {
+		if ((t.name ?: "").equals(""))
```

Comment:
(nitpick) no podemos ponerle la variable el nombre de `test`  en vez de `t`, entiendo que aca se ve a simple vista que es un t, pero seria mejor si le damos un nombre lindo

Reply:
Sí, tenés razón, y al principio estuve más decidido a hacer eso, había variables `x` y de otros nombres más raros. Pero el tiempo fue pasando y tuve que relajar eso, fijate que el WollokDslValidator está  muy lleno de eso: otra opción es renombrar la variable a `it` y directamente desaparecerla, quizás eso sea tentador y lo haga.